src/atcore.cpp

```diff
@@ -150,8 +151,22 @@ QList<QSerialPortInfo> AtCore::serialPorts() const
 void AtCore::newMessage(const QByteArray &message)
 {
     lastMessage = message;
+    if (message.startsWith(QString::fromLatin1("X:").toLocal8Bit())) {
+        posString = message;
+        posString.resize(posString.indexOf("E"));
+        posString.replace(":", "");
+    }
```

Comment:
the returned string from the printer is X:### Y:### Z:### E:#### the above is removing the E section since we do not care about the extruders position. Then we remove the remaining ":" so the args are ready to be sent later when resumed.